docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -145,9 +145,27 @@ the work and use the terms we find most accurate and inclusive.
 
 1.2 Glossary of General Terminology
 -----------------------------------
-In this section, we will create a glossary of general terminology that will be used 
-throughout this style guide (e.g., for starters, we should explicitly define sex and 
-gender; SGM) (include visual of 'Do's and Don'ts)
+In this section, you will find a glossary of general terminology that will be used 
+throughout this style guide. 
+
+
+.. list-table:: **Glossary**
+   :header-rows: 1
+
+   * - Term
+     - Definition
+   * - Sex
+     - *Sex* refers to biological attributes and functions associated with chromosomal makeup, endocrine function, reproductive organs, secondary sex characteristics, and other physiological and morphological variations and processes not directly related to reproduction (e.g., body size and composition, metabolic function, or organ function) [Ritz_and_Greaves_2022]_.
```

Comment:
It seems like these are very close paraphrases of the Ritz and Greaves paper; probably good to simplify a little bit and put in our own words, or just include the full quotes?

Reply:
Agreed, will try to simplify before merging